content/en/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog 
+title: Announcing the Auto-refreshing Official Kubernetes CVE Feed
+date: 2022-09-12 
+slug: k8s-cve-feed-alpha
+---
+
+**Author**: Pushkar Joglekar (VMware)
+
+A long-standing request from the Kubernetes community has been to have a
+programmatic way for end users to keep track of Kubernetes security issues
+(CVEs, named after the database that tracks public security issues across
+different products and vendors). Accompanying the release of Kubernetes v1.25,
+we are excited to announce availability of such
+a [feed](/docs/reference/issues-security/official-cve-feed/) as an `alpha`
+feature. This blog will cover the background, scope and details on how this
+feature was implemented.
+
+## Motivation
+
+With the growing number of eyes on Kubernetes, the number of CVEs related to
+Kubernetes have increased. Although most CVEs that directly, indirectly, or
+transitively impact Kubernetes are regularly fixed, there is no single place for
+the end users of Kubernetes to programmatically subscribe or pull the data of
+fixed CVEs. Current options are either broken or incomplete.
+
+## Scope
+
+### Goals
```

Comment:
Since this is for the main blog, consider using the more general-audience-friendly subtitles of:

### What This Does

### What It Doesn't Do

### Who It's For

Reply:
done